packages/services-store/src/types/translateAPI.ts

```diff
@@ -85,13 +85,52 @@ export interface MastheadLink {
 
 /**
  * A menu section for masthead
+ * @deprecated
  */
-export interface MastheadL1 {
+export interface LegacyMastheadL1 {
   title: string;
   url?: string;
   menuItems?: MastheadLink[];
 }
 
+/**
+ * A menu section for masthead
+ */
+export interface MastheadL1 {
+  title: string;
+  url?: string;
+  menuItems: L1MenuItem[];
+  actions?: {
+    cta?: BasicLink;
+    login?: BasicLink;
+  };
+}
+
+export interface L1MenuItem extends BasicLink {
+  submenu?: L1Submenu;
+}
+
+export interface L1Submenu {
+  announcement?: string; // From AEM rich text editor
+  menuSections: L1SubmenuSection[]; // maximum of 3 in outer array | renamed to be agnostic to the design
+  footer?: {
+    title: string;
+    url: string;
+  };
+}
+
+export interface L1SubmenuSection {
+  span: 1 | 2 | 3;
+  items: BasicLink | L1SubmenuSectionHeading[];
+}
```

Comment:
I'm still not sure about grouping links and heading as `items`. An argument can be made that this design shows two `L1SubmenuSections` in a single column layout, rather than an unstructured collection of headings and links:

<img width="763" alt="Screenshot 2022-12-08 at 10 40 39 AM" src="https://user-images.githubusercontent.com/12532727/207956347-38032e89-bef2-453e-9938-64d5a395542b.png">

In fact, looking at the designs again, I am feeling more and more like this is the paradigm we should be using, as they lay out specific one, two, and three column layouts. In section 4.3.4 they also mention the column span variation is "available only on the 3-column menu".

This leads me to think it would be wiser to establish `columnLayout: 1 | 2 | 3;` in the `L1Submenu` interface and change this to:

```suggestion
export interface L1SubmenuSection {
  span?: 1 | 2;
  items: BasicLink;
  heading: L1SubmenuSectionHeading[];
}
```

where `span` will only be consulted if the parent submenu has `columnLayout: 3`.